container/karaf/src/main/filtered-resources/etc/system.properties

```diff
@@ -161,6 +161,10 @@ karaf.secured.services = (&(osgi.command.scope=*)(osgi.command.function=*))
 # OPENNMS: Enable SNMP4J Log4j logging
 snmp4j.LogFactory=org.snmp4j.log.Log4jLogFactory
 
+# OPENNMS: Disable SIGTERM in karaf as OpenNMS should handle SIGTERM.
+karaf.handle.sigterm = false
```

Comment:
How will this affect karaf in Sentinel and Minion?

Reply:
afaik,  this properties file is specific to OpenNMS. It won't effect Sentinel/Minion as they are forked from [here](https://github.com/OpenNMS/opennms/tree/develop/features/container/sentinel/src/main/filtered-resources/etc)

Reply:
That's true. It is important to guarantee that this version of `system.properties` is *only* for OpenNMS, and that file won't be shared with Sentinel or Minion, as the Docker images for these 2 relies on having Karaf properly handling SIGTERM.